src/components/drawer/__internal__/drawer.stories.js

```diff
@@ -317,7 +317,7 @@ Visual.story = {
   name: 'visual',
   parameters: {
     chromatic: {
-      disable: false
+      disable: true
```

Comment:
We need this story, here we have rendered all possible states of Drawer. We could ignore the separate stories in `mdx` file instead.